client/apollo/js/View/Track/DraggableHTMLFeatures.js

```diff
@@ -1058,6 +1059,7 @@ var draggableTrack = declare( HTMLFeatureTrack,
         var fdiv = featdiv;
         while (fdiv.feature || fdiv.subfeature) {
             if (fdiv.parentNode.block) { return fdiv.parentNode.block; }
+            if (fdiv.parentNode.parentNode.block) { return fdiv.parentNode.parentNode.block; }
```

Comment:
@abretaud  this implementation seems fine, but

Reply:
I think overall fine, but:
1. in your comment it says `if inferHTMLSubfeatures is true`, but I don't see a check here
1. do you think there needs to be a null check for parentNode at all (didn't seem like there was one there before)? 
1. have you tested this with single-level features?   Not common for your users, but don't want to see an exception.